WrightTools/units.py

```diff
@@ -20,7 +21,7 @@
 angle = {"rad": ["x", "x", r"rad"], "deg": ["x/57.2958", "57.2958*x", r"deg"]}
 
 # delay units (native: fs)
-fs_per_mm = 3336.
+fs_per_mm = 3336.0
```

Comment:
This needs to be added to the numexpr locals, I think (at least wish to be explicit)

Reply:
In point of fact, it works, so just going to leave it for now, but being more explicit may be nice.